### PR TITLE
Vagrant file archived and updated to new placement

### PR DESCRIPTION
**Pull request by LeandroGuillen**
It still points to the script file, in case someone wants to use it from the archive directory. Also removed CoAP support.

### CODE REVIEW

File: archive/Vagrantfile [hunk 1]
```diff
@@ -34,10 +34,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "chef/centos-6.5"
```
Comment (fgalan) [inline archive/Vagrantfile]: Maybe is better to have an archive/vagrant instead that using the archive/ "root".

In addition, move the doc/manuals/vagrant.md to the archive (removing the CoAP reference in that file) and the link to it in the master README.md.

Comment (LeandroGuillen) [inline archive/Vagrantfile]: Fixed in 90192d8

Comment (LeandroGuillen) [inline archive/Vagrantfile]: Fixed in 90192d8 and 2a53f50

File: doc/manuals/vagrant.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ and run it from the command-line, or directly run the broker with something like
 
     vagrant ssh -c 'contextBroker -multiservice -t 0-255'
 
-Orion Context Broker will be accessible at `127.0.0.1:1026` (`5683` for CoAP).
```
Comment (fgalan) [inline doc/manuals/vagrant.md]: I understand that doc/manuals/vagrant.md should be removed, now that archive/vagrant/doc/manuals/vagrant.md exits.

File: archive/vagrant/doc/manuals/vagrant.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+After a few minutes you can run contextBroker. Again, you can either ssh into the machine with 
+
+    vagrant ssh
+
+and run it from the command-line, or directly run the broker with something like
+
+    vagrant ssh -c 'contextBroker -multiservice -t 0-255'
+
+Orion Context Broker will be accessible at `127.0.0.1:1026` (`5683` for CoAP).
```
Comment (fgalan) [inline archive/vagrant/doc/manuals/vagrant.md]: Reference to CAOP should be removed.

Comment (LeandroGuillen) [inline archive/vagrant/doc/manuals/vagrant.md]: Fixed in 79c08ec

File: doc/manuals/vagrant.md (hunk 2 above)
Comment (LeandroGuillen) [inline doc/manuals/vagrant.md]: Fixed in 79c08ec

**Comment by fgalan** [comment]
LGTM

**Pull request merged**